exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>Could you put the two verbs in the section header in the same form?</p><p> </p><p>Once we have a connection to the database we can run either the ALTER SESSION or the ALTER SYSTEM statement. The latter statement will activate the container permanently and globally. Both statements can be found in the output of the <code>generate_language_activation</code> function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Fixed</p>